stories/demos/components/Filters/NumericFilter.tsx

```diff
@@ -12,14 +12,14 @@ type Rule =
   | { type: RuleType.Range; begin: number; end: number }
   | { type: RuleType.GreaterThan | RuleType.LessThan | RuleType.Number; value: number };
 
-interface ChangeEvent<R> {
+interface ChangeEvent<R, SR> {
```

Comment:
Same here. We will probably have to change `TSummaryRow = any` in `FilterRendererProps`